admin/routes/service-manager.js

```diff
@@ -28,5 +28,10 @@ router.get('/mod-settings/cancel', isAuthenticated(roles.serviceManager), (req,
 router.get('/mod-settings/add-school', isAuthenticated(roles.serviceManager), (req, res, next) => serviceManagerController.getSceAddSchool(req, res, next))
 router.post('/mod-settings/add-school', isAuthenticated(roles.serviceManager), (req, res, next) => serviceManagerController.postSceAddSchool(req, res, next))
 router.get('/mod-settings/remove-school/:urn', isAuthenticated(roles.serviceManager), (req, res, next) => serviceManagerController.getSceRemoveSchool(req, res, next))
+router.get('/service-message', isAuthenticated(roles.serviceManager), (req, res, next) => serviceManagerController.getServiceMessage(req, res, next))
+router.get('/service-message/submit-service-message', isAuthenticated(roles.serviceManager), (req, res, next) => serviceManagerController.getSubmitServiceMessage(req, res, next))
```

Comment:
This is confusing the HTTP verb is GET but the URL has another verb called submit.   I find this a little confusing.

Reply:
@jon-shipley updated